cmake-edit-ui/src/main/java/ru/urururu/cmakeedit/ui/CmakeTextPane.java

```diff
@@ -144,4 +105,79 @@ public void changedUpdate(DocumentEvent e) {
     private void colorize(Node node, Style style) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this method into "CheckerWorker". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3398)